<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textkeypad</class>
 <widget class="QWidget" name="textkeypad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1437</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1471</width>
     <height>392</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/img/textkeypad1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_13">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>20</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>529</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>1284</x>
     <y>167</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/img/ENTER1.png);
background-color: rgba(191, 64, 64, 0);border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>1250</x>
     <y>20</y>
     <width>171</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_14">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_15">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_16">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_17">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_18">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_19">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_20">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_22">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_23">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>90</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_25">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_28">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_29">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_31">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>170</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_34">
   <property name="geometry">
    <rect>
     <x>729</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_35">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_36">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_37">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_38">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_39">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_40">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_41">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>240</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string>?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_42">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>641</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgba(191, 64, 64, 0);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
